misc/test-client-rust/src/util.rs

```diff
@@ -120,3 +116,29 @@ pub fn prompt_for_input(prompt: &str) -> String {
     std::io::stdin().read_line(&mut input).unwrap();
     input.trim().to_string()
 }
+
+pub fn validate_key_name(name: Option<&String>) -> &str {
```

Comment:
optional nit: `Option<&str>` is a more common type for optional string parameters. `.as_deref()` can be called on `Option<String>` instances to get an `Option<&str>`.